pages/signup.js

```diff
@@ -78,33 +84,116 @@ export default function Signup() {
-            <Input sx={{ my: .3 }} required={true} disableUnderline={true} onChange={(e) => {handleSignUpFormInput(e.target.value, 'password'), handlePasswordLength() }} placeholder="Password" className={classes.userInput} type="password"></Input>
-            <Typography sx={{fontWight: 'light', fontSize: 10, fontStyle: 'italic', color: passwordLengthColor }}  >*minimum password length of 6 characters</Typography>
-            <Button sx={{ my: 2 }} type='submit' className={classes.loginButton}>Create Account</Button>
-            <Button sx={{ my: 2 }} disabled={filledFormFlag} onClick={(e) => signInWithGoogle(signupInfo)} className={classes.googleButton} startIcon={<GoogleIcon />}>Continue with Google</Button>
-            <Button onClick={(e) => signInWithFacebook(signupInfo) } disabled={filledFormFlag} className={classes.facebookButton} startIcon={<FacebookIcon />}>Continue with Facebook</Button>
+            <Stack direction="row" spacing={1} sx={{my: 1}}>
+            <TextField size='small' placeholder='First Name' sx={{width:'50%', backgroundColor: '#F5F5F5'}} onChange={(e) => handleSignUpFormInput(e.target.value, 'firstname')}></TextField>
+            <TextField size='small' placeholder='Last Name' sx={{width:'50%', backgroundColor: '#F5F5F5'}} onChange={(e) => handleSignUpFormInput(e.target.value, 'lastname')}></TextField>
+            </Stack>
+            <TextField size='small' placeholder='Username' sx={{width:'100%', backgroundColor: '#F5F5F5'}}  onChange={(e) => handleSignUpFormInput(e.target.value, 'username')}></TextField>
+            <TextField size='small' placeholder='Email' sx={{width:'100%', backgroundColor: '#F5F5F5', my: 1}} onChange={(e) => handleSignUpFormInput(e.target.value, 'email')}></TextField>
+            <TextField size='small' placeholder='Location'  sx={{width:'100%', backgroundColor: '#F5F5F5', mb:1}} onChange={(e) => handleSignUpFormInput(e.target.value, 'location')}></TextField>
+            <TextField size='small' type='password' placeholder='Password' sx={{width:'100%', backgroundColor: '#F5F5F5', mb:1}} onChange={(e) => { handleSignUpFormInput(e.target.value, 'password'), handlePasswordLength() }}></TextField>
+            <Typography sx={{ fontWight: 'light', fontSize: 10, fontStyle: 'italic', color: passwordLengthColor }}  >*minimum password length of 6 characters</Typography>
+            <Button sx={{ my: 2 }} type='submit' className={classes.loginButton} disabled={filledFormFlag}>Create Account</Button>
+            <Modal open={googleOpen} onClose={handleGoogleClose}>
+              <FormControl sx={{
+                position: 'absolute',
+                alignItems: 'center',
+                top: '50%',
+                left: '50%',
+                transform: 'translate(-50%, -50%)',
+                width: 400,
+                bgcolor: 'background.paper',
+                borderRadius: 5,
+                boxShadow: 24,
+                p: 4,
```

Comment:
Abstract out styling from this `sx={...}` see comment below for more details

Reply:
![Screen Shot 2022-03-24 at 8 47 51 PM](https://user-images.githubusercontent.com/1490387/160050593-76352283-a025-4488-9e62-eb02bcaf7782.png)

Doesn't render the imported mui

Reply:
What if you included it as an object at the top of the page? If it doesn't work no worries, just curious